overlord/booted.go

```diff
@@ -57,22 +60,56 @@ func SyncBoot() error {
 	kernelSnap, _ := bootloader.GetBootVar("snappy_kernel")
 	osSnap, _ := bootloader.GetBootVar("snappy_os")
```

Comment:
Yes, we can, however I would like to create a trello card for this and track this separately. This involves changes to all the gadget snaps and to our initramfs package (and to snapd itself of course).